console/console-init/ui/src/constant/constants.ts

```diff
@@ -82,6 +82,11 @@ enum Messages {
   InvalidJson = "Invalid json, please try again with valid json"
 }
 
+enum PageJourney {
```

Comment:
please revisit this enum name.

Reply:
Purpose of this  enum is header change based on page so I thought to create enum as PageJourney which is indicate the page name. I feel this name is fine.